scripts/fixCodeCoverageStyle.py

```diff
@@ -1,5 +1,5 @@
 """
-Fixes VSTS coverage reports by inlining CSS files.
+Fixes coverage reports by inlining CSS files.
 see https://github.com/Microsoft/vsts-tasks/issues/3027
```

Comment:
Shouldn't we update it to the latest url - https://github.com/Microsoft/azure-pipelines-tasks/issues/3027